pyirf/benchmarks/angular_resolution.py

```diff
@@ -52,8 +55,17 @@ def angular_resolution(
     result[f"{energy_key}_center"] = 0.5 * (energy_bins[:-1] + energy_bins[1:])
     result["n_events"] = 0
 
-    key = "angular_resolution"
-    result[key] = u.Quantity(np.nan, table["theta"].unit)
+    # keep backwards compatibility
+    if not isinstance(quantile, list) and not isinstance(quantile, np.ndarray):
```

Comment:
`isinstance` supports tuples for testing multiple classes:
```
isinstance(quantile, (list, ndarray))
```
but better would probably just to test for `Sequence`